### PR TITLE
Odsc 53443/eval/get eval metrics md

### PR DESCRIPTION
**Pull request by mingkang111**
# Description
JIRA: https://jira.oci.oraclecorp.com/browse/ODSC-53443

Gets evaluation metrics from evaluation artifact. We expect `report.json` in evaluation artifact for metrics metadata and metric markdown (e.g. `metric_key.md`) in evaluation artifact. 

# Changed 
* implemented `AquaEvaluationApp.load_metrics`
* Added test cases

# Test
* Unittest
Screenshot 2024-02-27 at 5 05 17 PM

* Integration test
```
http://localhost:8888/aqua/evaluation/<EVAL_OCID>/metrics
```

### CODE REVIEW

File: ads/aqua/evaluation.py [hunk 1]
```diff
@@ -674,23 +676,67 @@ def load_metrics(self, eval_id: str) -> AquaEvalMetrics:
...
+                        name=v.get("name", utils.UNKNOWN),
+                        content=content,
+                        description=v.get("description"),
                     )
-        return AquaEvalMetrics(id=eval_id, metrics=metrics)
+                )
+
+        eval_metrics = AquaEvalMetrics(id=eval_id, metrics=metrics)
+
+        self._metrics_cache.__setitem__(key=eval_id, value=eval_metrics)
```
Comment (VipulMascarenhas) [inline ads/aqua/evaluation.py:737]: can metrics remain empty when load_metrics is called as evaluation is in progress? In that case, should we check that before we cache?

**Review by VipulMascarenhas — APPROVED** [review]
added a comment, rest looks good.

File: ads/aqua/evaluation.py (hunk 1 above)
Comment (mingkang111) [reply ads/aqua/evaluation.py:744]: Added validation for content.

Comment (mingkang111) [reply ads/aqua/evaluation.py]: Added validation of content before setting cache.

**Pull request merged**